cli/cmd/root.go

```diff
@@ -226,6 +227,7 @@ type proxyConfigOptions struct {
 	debugImageVersion             string
 	dockerRegistry                string
 	imagePullPolicy               string
+	destinationGetNetworks        []string
```

Comment:
I think for now we should omit this from the CLI flags, leaving it only as an annotation or helm setting.

Reply:
Okey, we can do that. Would that mean that we do not want it in the config map or this is not related. @alpeb WDYT ?

Reply:
I think it would be preferable to omit it, but I think it's ultimately an implementation detail.